microprofile-fault-tolerance/pom.xml

```diff
@@ -150,7 +150,12 @@
         </dependency>
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
-            <artifactId>resteasy-jaxrs</artifactId>
+            <artifactId>resteasy-core-spi</artifactId>
```

Comment:
It it's more critical to get the release out this isn't a huge deal, but this dependency can go away and in the `org.wildfly.quickstarts.microprofile.faulttolerance.Coffee.coffeeDetail()` the `@PathParam` import can change to `javax.ws.rs.PathParam` which is more appropriate IMO.

Reply:
done, and works, thanks